pkg/tfshim/sdk-v2/provider.go

```diff
@@ -54,30 +47,35 @@ func diffToShim(d *terraform.InstanceDiff) shim.InstanceDiff {
 type v2Provider struct {
 	tf   *schema.Provider
 	opts []providerOption
+
+	server       *grpcServer
+	planEditFunc PlanStateEditFunc
+
+	// All resources to serve schema.
+	resources map[string]*schema.Resource
```

Comment:
Self-review: this may be no longer populated, need to restore how it was populated.

Reply:
Ah this is just always self.tf.ResourceMap.